testproto/pool/pool_with_slice_reuse_vtproto.pb.go

```diff
@@ -8,6 +8,7 @@ import (
 	fmt "fmt"
 	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 	io "io"
+	bits "math/bits"
```

Comment:
changes in this file do not look relevant, can revert if you prefer to commit it separately at later point of time